### DISCUSSION TITLE
possible to connect using tableau？

### DISCUSSION THREAD

**Discussion by beswarm** [duckdb/duckdb#2366] [Q&A] [answered]
I download the duckdb jdbc driver and copy it to the install directory: C:\Program Files\Tableau\Drivers\duckdb_jdbc-0.2.9.jar then I start the tableau , and choose the others jdbc drivers to connect, set the configuration and try connect to duckdb，it t fail.  I already checkout the offical documents and found nothing related to this.

by the way, I can connect successfully using the Jetbrains Datagrip IDE after set the proper className and the same connection url?

**Comment by Mause** [comment]
What error do you get?

**Reply by beswarm** [reply]
the configuration:
![image](https://user-images.githubusercontent.com/7619685/135741536-e452522e-e311-4af4-b551-56062c489fdf.png)

the error screen:
![image](https://user-images.githubusercontent.com/7619685/135741555-6495c29d-4c32-425b-9d73-08782d7f6ba0.png)

**Comment by Raidus** [comment]
I am facing the similar issue on my Mac M1 (macOS v12.1) with Tableau 2021.4, using the latest jdbc driver `duckdb_jdbc-0.4.0.jar` which is placed in `/Library/Tableau/Drivers`.

![image](https://user-images.githubusercontent.com/8803882/177836495-ace43031-6b86-4f37-8dbd-5586a6d69a58.png)

Using the same jdbc driver version I can connect with DBeaver without problems. Might be an issue with Tableau only?

**Comment by erikhuck** [comment]
@beswarm I figured out how to connect to Tableau using the DuckDB JDBC. Are you still having issues?

**Comment by hawkfish** [comment]
These days I strongly recommend using [my connector](https://github.com/hawkfish/duckdb-taco). I have used it successfully with MacOS and a Linux Tableau Server. Please let me know if you have any problems with it.

Also, you should use the latest JDBC driver if possible. There have been a few post 0.8.1 fixes for time stamps.

**Comment by hawkfish** [comment, marked as answer]
We have [updated](https://duckdb.org/docs/guides/data_viewers/tableau) the documentation so please leet us know if you have any further issues.
